build/deb_rpm/build-scripts/pack.sh

```diff
@@ -6,8 +6,8 @@
 # Although by passing a different --rpm-build-image or --rpm-build-image, artifacts of different architectures can be built
 SUPPORTED_DEFAULT_ARCH_LIST="x86_64"
 
-JFROG_CLI_HOME="$(cd "$(dirname "${BASH_SOURCE[0]}")/../" && pwd)"
-JFROG_CLI_PKG="$JFROG_CLI_HOME/pkg"
+JFROG_CLI_HOME_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")/../" && pwd)"
```

Comment:
This file should not be modified, because the ```JFROG_CLI_HOME``` variable used here is not the ```JFROG_CLI_HOME``` environment varoable used by jfrog-cli. This is a private (internal) variables that this bash script defines. This script builds the rpm and debian packages for jfrog-cli.